content/resources/operator/_index.md

```diff
@@ -25,20 +25,23 @@ Cloud automation is how we deploy Gen3 data commons in production environments o
+Gen3 introduced the [DCF data dictionary](https://github.com/uc-cdis/dcfdictionary) that allows users to construct their own data dictionary. It could serve as a starting point for someone who is interested in creating their own dictionary. It's a consensus of previously used data dictionaries and makes the process of creating a data dictionary more efficient.  It's composed of several nodes that are categorized based on the type of data they are modeling such as Clinical, Biospecimen, and Data File.  The Project, Study, and Subject nodes are administrative nodes that are required for any Gen3 data commons.  Administrative nodes store basic project and study information for their associated cases or subjects.  Also, the Subject node (which is also an administrative node) level is where the nodes start differentiationg between commons.  It can also be represented as Case depending on the use case.  The Subject node links to the Demographic and Diagnosis clinical nodes.  These nodes are Clinical nodes that are used to store clinical and medical history related data.  The Demograpic node stores properties that represent the statistical characterization of human populations or segments of human populations (for example, characterization by year of birth, sex, and race).  This node is typically used to store properties that do not change over time.  The Diagnosis clinical node represents the investigation, analysis, and recognition of the presence and nature of disease, condition, or injury from expressed signs and symptoms. It also pertains to the scientific determination of any kind and the concise results of such an investigation.  A clinical node that is not included in the DCF is the Visit or Follow-Up node.  The Visit node is used to store longitudinal data that is collected over time and usually has a many to one relationship with its parent node.  Another node category is the index file that stores the metadata that is associated with different file formats (forexample, BAI and BAM).  The Analysis node category stores data that is associated with genomic pipline data that is typical of next generation sequencing (NGS).  Finally, the Notation node category is used to store data that does not fit into other categories (for example, it doesn't store index files, data files, or analysis data).  The Data File node category is used to store metadata related to data files that are stored in the cloud.  If the need arises, the node can be added to a data dictionary.  The ability to update/modify a dictionary is an important functionality that may arise based on project and clinical data needs. 
```

Comment:
misspelled differentiating 

space between for and example here: (forexample, BAI and BAM)